solution.cpp

```diff
@@ -1,23 +1,121 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
+ * Copyright &copy; 2015 Venedicto Herrera venedicto@csu.fullerton.edu
  * Released under the [MIT License] (http://opensource.org/licenses/MIT)
  * ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * convert a given int (assuming input is valid) to a Roman Numeral, repeat option via user input
  */
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+#include <string>
 
-// ----------------------------------------------------------------------------
+using std::cin;
+using std::cout;
+using std::endl;
 
-int main() {
-    // TODO
+std::string convertToRoman();
+int num;
```

Comment:
ahh... i see what you were doing.  `num` should probably have been passed as an  argument to `convertToRoman`, rather than being global -- safer that way, in general, and especially in more complex projects